uwblueprint/project-read-frontend
# Refactor family form components to reuse in the sidebar (1/n)

### Notion ticket link



[Display the family form in the sidebar](https://www.notion.so/uwblueprintexecs/Developer-Hub-30e9b6c82af24054b31acf7c5e822c89?p=339f42db2c08477ea9a29fb6e419b198)



### Implementation description

- no UX changes
- refactors the family form components to make it reusable for the sidebar
- essentially makes it so we have two reusable components: FamilyParentFields (parent & family fields) and StudentForm (children & guest fields)



### Steps to test

1. yarn test (RegistrationForm test should still pass)
2. open the form and verify behaviour is unchanged



### What should reviewers focus on?

- code quality

### Checklist
